src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/adaptive_pooling.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+using adapoolParams = std::tuple<
+        std::vector<InputShape>,            // feature map shape
+        std::vector<int>,                   // pooled spatial shape
+        std::string,                        // pooling mode
+        ov::element::Type,                  // net type
```

Comment:
@iefode @olpipi the test has only one "model" type, input and output types are implicitly "undefined". Is it needed to add types explicitly as it's done in other tests (https://github.com/openvinotoolkit/openvino/pull/19597/files#diff-cc3700c4244ea84f102cd6ff2a47b78daf6054fb35d761131c0a8bf50594cc37R13-R15)?

Reply:
I cannot answer what option is correct. @iefode @ilyachur, Could you please help?

Reply:
@olpipi NetType is absolutelu extra as I can see, we can define only input types as my opinion

Reply:
Changed to usage inType and outType